packages/ui/src/composites/wui-list-accordion/index.ts

```diff
@@ -0,0 +1,112 @@
+import { html, LitElement } from 'lit'
+import { property, state } from 'lit/decorators.js'
+import '../../components/wui-icon/index.js'
+import '../../components/wui-text/index.js'
+import '../../layout/wui-flex/index.js'
+import { elementStyles, resetStyles } from '../../utils/ThemeUtil.js'
+import { customElement } from '../../utils/WebComponentsUtil.js'
+import styles from './styles.js'
+
+const MAX_HEIGHT = 100
+
+@customElement('wui-list-accordion')
+export class WuiListAccordion extends LitElement {
+  public static override styles = [resetStyles, elementStyles, styles]
+
+  // -- State & Properties -------------------------------- //
+  @property() public textTitle = ''
+
+  @property() public overflowedContent = ''
+
+  @state() public toggled = false
+
+  @state() public enableAccordion = false
+
+  @state() public scrollElement?: Element = undefined
+
+  @state() public scrollHeightElement = 0
+
+  public override firstUpdated() {
+    setTimeout(() => {
+      const heightElement = this.shadowRoot?.querySelector('.heightContent')
+
+      if (heightElement) {
+        this.scrollElement = heightElement
+        const scrollHeight = heightElement?.scrollHeight
+        if (scrollHeight && scrollHeight > MAX_HEIGHT) {
+          this.enableAccordion = true
+          this.scrollHeightElement = scrollHeight
+        }
+      }
+    }, 0)
+  }
+
+  // -- Render -------------------------------------------- //
+  public override render() {
+    return html`
+      <button ontouchstart @click=${() => this.onClick()}>
+        <wui-flex justifyContent="space-between" alignItems="center">
+          <wui-text variant="paragraph-500" color="fg-100"> ${this.textTitle}</wui-text>
```

Comment:
```suggestion
          <wui-text variant="paragraph-500" color="fg-100">${this.textTitle}</wui-text>
```

Not sure if affects the UI but should avoid empty spaces imo